recipes/r-qgg/meta.yaml

```diff
@@ -0,0 +1,126 @@
+{% set version = '1.1.6' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-qgg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qgg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qgg/qgg_{{ version }}.tar.gz
+  sha256: dff06a1f4684622a4e60f617758254d25210f5081d814bc143906cea8b297a19
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+#  ignore_run_exports:
+#    - r-data.table
+#    - r-corpcor
+#    - r-rcpparmadillo
+#    - r-matrix
+#    - r-rcpp
+#    - r-statmod
+#    - r-mass
+#    - r-mcmcpack
+#    - r-coda
+
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ stdlib("c") }}            # [not win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ stdlib("m2w64_c") }}      # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - libopenblas 
+    - r-base
+    - r-mass
+    - r-mcmcpack
+    - r-matrix
+    - r-rcpp >=1.0.5
+    - r-rcpparmadillo
+    - r-coda
+    - r-corpcor
+    - r-data.table
+    - r-statmod
+
+  run:
+    - libopenblas
+    - m2w64-gcc-libgfortran  # [win]
```

Comment:
```suggestion
```
This should be added automatically by `{{ compiler('m2w4_fortran') }}` - let's remove it and see what happens.